inst/csv/replacementPatterns.csv

```diff
@@ -567,6 +571,9 @@ netezza,UPDATE STATISTICS @a;,GENERATE STATISTICS ON @a;
 netezza,DATETIME,TIMESTAMP
 netezza,DATETIME2,TIMESTAMP
 netezza,"ISNUMERIC(@a)","CASE WHEN translate(@a,'0123456789','') in ('','.','-','-.') THEN 1 ELSE 0 END"
+netezza,"HASHBYTES('MD5',@a)","hash(@a)"
+netezza,"CONVERT(VARBINARY, '0x' || @a, 1)","hex_to_binary(@a)"

```

Comment:
Why have you specified `||` here?

Reply:
The previous Netezza rules converts `CONCAT` into `||`. 
If leave expression as it comes to `translate` it would be position-sensitive.